webv2/primarykey/primarykey_handler.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package web defines web APIs to be used with harbourbridge frontend.
+// Apart from schema conversion, this package involves API to update
+// converted schema.
+
+package primarykey
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"github.com/cloudspannerecosystem/harbourbridge/spanner/ddl"
+	helpers "github.com/cloudspannerecosystem/harbourbridge/webv2/helpers"
+	"github.com/cloudspannerecosystem/harbourbridge/webv2/session"
+
+	"github.com/google/uuid"
+)
+
+// PrimaryKeyRequest represents  Primary keys API Payload.
+type PrimaryKeyRequest struct {
+	TableId      int      `json:"TableId"`
+	Columns      []Column `json:"Columns"`
+	PrimaryKeyId int      `json:"PrimaryKeyId"`
+}
+
+// PrimaryKeyResponse represents  Primary keys API response.
+// Synth is true is for table Primary Key Id is not present and it is generated.
```

Comment:
// PrimaryKeyResponse represents primary key API response.
// Synth is true for tables in which primary key is not present and is generated.